components/event-publisher-proxy/cmd/event-publisher-proxy/main.go

```diff
@@ -1,42 +1,43 @@
 package main
 
 import (
-	"k8s.io/client-go/dynamic"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-
 	"github.com/kelseyhightower/envconfig"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/cmd/event-publisher-proxy/beb"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/cmd/event-publisher-proxy/nats"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/metrics"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/options"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
+)
 
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/application"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/env"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/handler/http"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/informers"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/legacy-events"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/metrics"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/oauth"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/options"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/receiver"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/sender"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/signals"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/subscribed"
+const (
+	BEBBackend  = "beb"
+	NATSBackend = "nats"
```

Comment:
Does it make sense to not export those constants outside of package?

Reply:
yes, changed